web/html/src/manager/admin/access-control/access-group-permissions.tsx

```diff
@@ -120,11 +128,20 @@ const AccessGroupPermissions = (props: Props) => {
             props.onChange(initialChanges);
           }
         }
+        setIsLoading(false);
       })
       .catch(() => {
+        setIsLoading(false);
         showErrorToastr(t("An unexpected error occurred while fetching namespaces."));
       });
-  }, []);
+  }
+
+  useEffect(() => {
+    if (!isLoading) {
+      setIsLoading(true);
+      getNamespaces(searchValue);
+    }
+  }, [searchValue]);
```

Comment:
In this kind of search-as-you-type task, it's a good practice to limit the requests to the backend to prevent clogging up the network.

We already do this using the `debounce` utility from lodash. It wraps the method that does the API call and it waits for some milliseconds of inactivity before it calls the method. This way you avoid calling the API for every keystroke, and only do one call after the user pauses typing.

Here's an example usage (note the last arg: 100ms of wait time):

https://github.com/uyuni-project/uyuni/blob/fa55769c56d76af2ef586f972e47f9e8b2833ce2/web/html/src/manager/content-management/shared/components/panels/sources/channels/channels-selection.tsx#L83-L103

I think it should work if you wrap the `getNamespace` method like this and call it in `useEffect`.